README.md

```diff
@@ -0,0 +1,125 @@
+# KCP Helm Charts
+
+Repository for KCP helm charts.
+
+## Pre-requisites
+
+- Cert-manager installed and running
+- Ingress installed (e.g. nginx-ingress or OpenShift router)
+
+## Usage
+
+[Helm](https://helm.sh) must be installed to use the charts.  Please refer to
+Helm's [documentation](https://helm.sh/docs) to get started.
+
+Once Helm has been set up correctly, add the repo as follows:
+
+    helm repo add kcp https://kcp-dev.github.io/helm-charts
+
+If you had already added this repo earlier, run `helm repo update` to retrieve
+the latest versions of the packages.  You can then run `helm search repo
+kcp` to see the charts.
+
+To install the kcp chart:
+
+    helm install my-kcp kcp/kcp
+
+To uninstall the chart:
+
+    helm delete my-kcp
+
+## Development usage
+
+To install using the local chart:
+
+    helm install kcp ./charts/kcp --values ./myvalues.yaml --namespace kcp --create-namespace
+
+Changes can then be made locally and tested via upgrade:
+
+    helm upgrade kcp ./charts/kcp --values ./myvalues.yaml --namespace kcp
+
+Note `myvalues.yaml` will depend on your environment (you must specify which ingress method
+is used to expose the front-proxy endpoint), a minimal example:
+
+    externalHostname: "<external hostname as exposed by ingress method below>"
+    kcpFrontProxy:
+      ingress:
+        enabled: true
+
+## Accessing the deployed KCP
+
+To access the deployed KCP, it will be necessary to create a kubeconfig which connects via the
+front-proxy external endpoint (specified via `externalHostname` above).
+
+The content of the kubeconfig will depend on the KCP authentication configuration, below we describe
+one option which uses client-cert auth to enable a kcp-admin user.
+
+:warning: this example allows global admin permissions across all workspaces, you may also want to
+consider using more restricted groups for example `system:kcp:workspace:access` to provide a
+user `system:authenticated` access to a workspace.
```

Comment:
Nit: I don't think we should hardcode these kinds of groups (`system:kcp:workspace:access` will grant access to all workspaces automatically, no?) into certificates and instead link to [the authZ docs](https://docs.kcp.io/kcp/main/concepts/authorization/) to give users the tools to set up proper RBAC for their certificate-based users. So I would update this section with a brief note you can pass `commonName` in your cert instead of a system group and give permissions via RBAC to that new user. WDYT?

Reply:
Something to fix later, probably.